frontend/src/modules/LoginRedux.js

```diff
@@ -1,14 +1,81 @@
 // Action Types
 export const SET_LOGIN_STATE = "localize/SET_LOGIN_STATE";
+// URL
+export const BACKENDURL = "localhost";
+// AUTH ACTIONS
+export const AUTHENTICATED = "AUTHENTICATED";
+export const UNAUTHENTICATED = "UNAUTHENTICATED";
+export const TESTING_TOKEN_REFRESH = "TESTING_TOKEN_REFRESH";
+export const REGISTRATION_SUCCESS_MESSAGE = "REGISTRATION_SUCCESS_MESSAGE";
+//CHANGE PASSWORD ACTIONS
+export const IS_CHANGING_PASSWORD = "IS_CHANGING_PASSWORD";
+export const CHANGE_PASSWORD_SUCCESS = "CHANGE_PASSWORD_SUCCESS";
+export const CHANGE_PASSWORD_FAILURE = "CHANGE_PASSWORD_FAILURE";
 
 // Action Creators
 const setLoginState = loggedIn => ({ type: SET_LOGIN_STATE, loggedIn });
 
+let url = process.env.REACT_APP_DEV_URL || BACKENDURL;
+
+function authenticateAction(userData, dispatch, location, push) {
+  return async function() {
+    if (navigator.cookieEnabled) {
+      localStorage.setItem("ecom_token", userData.token);
+    }
+
+    if (location === "/login") {
+      push("/");
+    }
+    return dispatch({ type: AUTHENTICATED });
+  };
+}
+function registrationSuccessMessage() {
+  return { type: REGISTRATION_SUCCESS_MESSAGE };
+}
+
+function registerAction(data) {
+  return async function() {
+    let response = await fetch(`${url}/auth/users/create/`, {
```

Comment:
We need the url in order to access these APIs.
To access _swagger_, you need to use the following link: http://10.0.75.1:8000/

Then to access APIs, you can call it by using the following link for example: http://10.0.75.1:8000/auth/users/

This will allow you to get the users list, as shown below with **Postman**.

![image](https://user-images.githubusercontent.com/23021242/57528053-43216600-72ff-11e9-8cc0-4c69cfd4c30c.png)

![image](https://user-images.githubusercontent.com/23021242/57528041-3b61c180-72ff-11e9-89bf-f304ba2d17df.png)

See PR #279 for more details.